src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readData(fromFileName);
+        String report = createReport(data);
+        writeData(report, toFileName);
+    }
+
+    public String readData(String fromFileName) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
are resources closed?

Reply:
I don't know why, but i make it closed and this does not show it in changes